values.html

```diff
@@ -0,0 +1,188 @@
+
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>If These Lands Could Talk - Our Values</title>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link rel="stylesheet" href="css/global.css" />
+		<!-- In a new branch, rename this page.css file to the name of that actual page (Example: home.css) -->
+		<link rel="stylesheet" href="css/values.css" />
+	</head>
+
+	<body>
+		<!-- Header section -->
+		<!-- add aria labels -->
+		<section class="hero">
+			<div>
+				<!-- <source media="(min-width:768px)" srcset ="./assets/images/pages/values/hero-values.jpg" alt="a young man smiling"> -->
+				<!-- Not sure if desktop img needs new line  -->
+				<img
+					src="./assets/images/pages/values/mobile/hero-values-m.jpg"
+					alt="a young man smiling"
+				/>
+			</div>
+			<div class="values-main">
+				<div class="values-head">
+					<h3 class="h1-black">Our Values</h3>
+					<p>
+						We believe in equality and belonging for native and indigenous
+						communities across the world. We also believe that through these
+						values is where we can find an authentic future with digital
+						technology.
+					</p>
+				</div>
+			</div>
+		</section>
+		<!-- Values-list section -->
+		<div class="desktop-parent">
+			<!-- Desktop Group 1 START -->
+			<div class="desktop-group">
+				<section class="copperback values-card">
+					<h4 class="h2">Humanity and Inclusion</h4>
+					<p>
+						We believe in human rights, equity, inclusion, and fostering
+						belonging for all people to include native and indigenous
+						communities.
+					</p>
+				</section>
+				<section class="sustainability values-card onyback">
+					<img
+						src="./assets/images/pages/values/mobile/sustainability-m.jpg"
+						alt="a person holding a tablet in the middle of nature"
+					/>
+					<div class="sustainability-text">
+						<h4 class="h2">Sustainability</h4>
+						<p>
+							We support long term sustainability and responsible treatment of
+							our natural environment and its resources.
+						</p>
+					</div>
+				</section>
+				<!-- Desktop Group 1 END -->
+			</div>
+			<div class="desktop-group">
+				<section class="authenticity values-card">
+					<div class="authenticity-text">
+						<h4 class="h2">Authenticity</h4>
+						<p>
+							We believe in authentic representation and utility for indigenous
+							communities.
+						</p>
+					</div>
+					<img
+						src="./assets/images/pages/values/mobile/authenticity-m.jpg"
+						alt="an indigenous family laughing together"
+					/>
+				</section>
+
+				<section class="grapeback values-card">
+					<h4 class="h2">Multigenerational Connection</h4>
```

Comment:
Please double-check the spacing between the heading and paragraph on these cards.

![Uploading Screen Shot 2024-01-19 at 11.17.03 AM.png…]()